src/io.cr

```diff
@@ -679,7 +679,7 @@ abstract class IO
       return string
     end
 
-    # We didn't find the delimiter, so we append to a String::Builde
+    # We didn't find the delimiter, so we append to a String::Builder
```

Comment:
Could you also put `String::Builder` inside backticks?

Reply:
It's possible, but useless IMO, as it's a code comment, not a documentation comment.